File: Programming Basics/Java/Conditional Statements/GodzillaVsKong.java
package com.company;

import java.util.Scanner;

public class GodzillaVsKong {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double budget = Double.parseDouble(scanner.nextLine());
        int countStatists = Integer.parseInt(scanner.nextLine());
        double clothesPriceforOne = Double.parseDouble(scanner.nextLine());
        double dechorPrice = budget * 0.1;
        double clothesPrice = clothesPriceforOne * countStatists;
        if (countStatists > 150) {
            clothesPrice = (clothesPriceforOne * countStatists) * 0.9;
        }
        double totalExpenses = dechorPrice + clothesPrice;
        if ( budget >= totalExpenses) {
            System.out.printf("Action!%nWingard starts filming with %.2f leva left.", (budget - totalExpenses));
        }else {
            System.out.printf("Not enough money!%nWingard needs %.2f leva more.", Math.abs(budget - totalExpenses));
        }
    }
}

File: Programming Basics/JavaScript/Conditional Statements/Godzilla vs. Kong.js
function film(input) {
    let budget = Number(input.shift());
    let statisti = Number(input.shift());
    let oblekloCenaBroi = Number(input.shift());
    let dekorCena = budget * 0.10;
    let oblekloCena = statisti * oblekloCenaBroi;
    if (statisti > 150) {
        oblekloCena = oblekloCena * 0.9;
    }
    let obshto = dekorCena + oblekloCena;
    let Money = Math.abs(budget - obshto);
    if (budget >= obshto) {
        console.log("Action!");
        console.log(`Wingard starts filming with ${Money.toFixed(2)} leva left.`);
    } else {
        console.log("Not enough money!");
        console.log(`Wingard needs ${Money.toFixed(2)} leva more.`)
    }
}
film([9587.88,
    222,
    55.68,
])